department-of-veterans-affairs/abd-vro
# Move 1.0 end points out of demo.

# Move v1.0 End Points Out of Demo Packages

## Description



### What was the problem?

V1.0 End points were implemented in demo packages. Now that these end points are is closer to final they are moved out of demo packages to parents. Also updated swagger documentation. One end point is left in the demo packages; looks like that end point is still being used by one of the python services. I also left those demo packages in progress implementations that we might want to demo.

### How does this fix it?

Moves java artifacts in demo packages to their parent package

### Jira Tickets

- [MCP-1592](https://amida.atlassian.net/browse/MCP-1592)

## How to test this PR

- Open Open API doc (http://localhost:8080/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config)
- Looks at overall documentation. - starter api is still there. Those will be removed next iteration.
- ABD-VRO API section should now include the three end points.
- Try out the end points
- ABD-VRO Demo API should be separate

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
